src/lib.rs

```diff
@@ -197,11 +197,11 @@ impl RenderPass {
             label: Some("egui_pipeline"),
             layout: Some(&pipeline_layout),
             vertex: wgpu::VertexState {
-                entry_point: if output_format.is_srgb() {
+                entry_point:Some( if output_format.is_srgb() {
```

Comment:
Could you please run cargo fmt over your PR?

Reply:
`cargo fmt` was unable to format the code.(executed but no changes were made)
I formatted it by hand.